### PR TITLE
update IsSignerExist for sm2

### PR DESCRIPTION
**Pull request by Frankonly**
#1925

### CODE REVIEW

File: ioctl/cmd/account/account.go [hunk 1]
```diff
@@ -178,6 +178,29 @@ func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
...
+	for _, ksAccount := range ks.Accounts() {
+		if address.Equal(addr, ksAccount.Address) {
+			return true
+		}
+	}
+	// find the account in pem files
+	pemFilePath := sm2KeyPath(addr)
+	_, err = os.Stat(pemFilePath)
+	if err == nil {
+		return true
```
Comment (dustinxie) [inline ioctl/cmd/account/account.go]: return (err == nil)?

Comment (dustinxie) [reply ioctl/cmd/account/account.go]: also, i recall seeing this code (check KeyStore then PEM file) in your SM2 check-in

why it is missed

Comment (Frankonly) [reply ioctl/cmd/account/account.go]: I used to put all account related functions in account.go. I don't why someone put this function in action.go, which is odd and I missed it last time. Now, I have put this function in account.go.

File: ioctl/cmd/account/account.go [hunk 2]
```diff
@@ -178,6 +178,25 @@ func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
...
 
+// IsSignerExist checks whether signer account is existed
+func IsSignerExist(signer string) bool {
+	addr, err := address.FromString(signer)
+	if err != nil {
+		return false
+	}
+	// find the account in keystore
+	ks := keystore.NewKeyStore(config.ReadConfig.Wallet,
+		keystore.StandardScryptN, keystore.StandardScryptP)
```
Comment (CoderZhi) [inline ioctl/cmd/account/account.go]: format

File: ioctl/cmd/account/account.go [hunk 3]
```diff
@@ -178,6 +178,25 @@ func GetAccountMeta(addr string) (*iotextypes.AccountMeta, error) {
...
+	for _, ksAccount := range ks.Accounts() {
+		if address.Equal(addr, ksAccount.Address) {
+			return true
+		}
+	}
+	// find the account in pem files
+	_, err = findSm2PemFile(addr)
```
Comment (CoderZhi) [inline ioctl/cmd/account/account.go]: I still don't see why we need a mixture of sm2 and current pems. Sm2 account format is not determined by user input, https://github.com/iotexproject/iotex-core/blob/1e3895b75321f0ac8b946621958923c08d518fac/ioctl/cmd/account/accountcreateadd.go#L47. "xctl" always and only creates or uses sm2 accounts.

Overall, I think you implementation is misleading to users of "xctl", who shouldn't be aware of the exists of two different account systems or algorithms.

**Pull request merged**